scripts/post-install.sh

```diff
@@ -90,12 +90,17 @@ elif [[ -f /etc/os-release ]]; then
     source /etc/os-release
     if [[ $ID = "amzn" ]]; then
         # Amazon Linux logic
-        install_init
-        # Run update-rc.d or fallback to chkconfig if not available
-        if which update-rc.d &>/dev/null; then
-            install_update_rcd
+        if [[ $VERSION_ID = "2" ]]; then
```

Comment:
I expect that when VERSION_ID=3 is released it will still be using systemd, so maybe we can have the systemd path be default and only do the other methods for VERSION_ID=1 (if there is such a variable)

Reply:
Good point, I'll switch this around.